test/acceptance/keywords/run_on_failure_only_once.txt

```diff
@@ -0,0 +1,30 @@
+*** Settings ***
+Suite Setup     Run Keywords  Prefer Custom Keywords  Open Browser To Front Page
+Suite Teardown  Run Keywords  Close Browser  Set Debug Loglevel  Restore Old Search Order
+Test Teardown   Register Keyword To Run On Failure  Nothing
+Resource        ../resource.txt
+Library         CustomSeleniumKeywords
+
+*** Test Cases ***
+Run On Failure Keyword Only Called Once
```

Comment:
The way this test is setup it looks like there's a keyword called ```Run On Failure Only Once```. My recommendation is to add this test case to ```test/acceptance/keywords/run_on_failure.txt``` since it actually tests that keyword. This doesn't need to be in its own file.